app/enterprise/0.31-x/index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: Kong Enterprise Edition (EE) Documentation
+title: Kong Enterprise Documentation
 ---
-# Welcome to Kong Enterprise Edition
+# Welcome to Kong Enterprise
 
 {% if page.edition == 'enterprise' %}
 <div class="alert alert-ee">
   <div class="alert-body">
     <div class="left">
       <img src="/assets/images/icons/icn-buildings.svg" />
     </div>
-    <p>Kong Enterprise Edition (EE) adds features, functionality, and performance to Kong Community Edition (CE). This documentation doesn’t cover the general practices of using Kong that are common to both Kong CE and EE - learn the <a href="">basics in Kong CE documentation</a>.</p>
+    <p>Kong Enterprise adds features, functionality, and performance to Kong. This documentation doesn’t cover the general practices of using Kong that are common to both Kong and EE - learn the <a href="">basics in Kong documentation</a>.</p>
```

Comment:
EE is mentioned.

I'll stop my review here and await a revised PR, thanks!

Reply:
I _think_ you could resolve most of the remaining EE mentions by searching for "EE" and replacing it with "Kong Enterprise" - but you'd want to review those as you go, just to be sure.